backend/models/item.js

```diff
@@ -1,19 +1,10 @@
-const mongoose = require("mongoose");
-const Schema = mongoose.Schema;
-
-const BatchSchema = new Schema({
-  date: Date,
-  pounds: Number,
-  fulfilled: Boolean,
-});
+const { Schema, model } = require("mongoose");
 
 const ItemSchema = new Schema({
   name: { type: String, unique: true },
   itemId: { type: String, unique: true },
-  batches: [BatchSchema],
```

Comment:
I'm a little confused, don't we still want a list of batches in the ItemSchema, so that we keep the batches of the same items together? Is there a reason we don't keep the corresponding batches nested inside of the ItemSchema anymore?

Reply:
I updated the `getAllItems` route, so the batches are still nested inside the items when they are returned by the route (in other words, the route behaves the same way). From a database perspective, having items in batches in separate collections made it easier to enforce uniqueness constraints (one batch per item and out date) and keep the batches sorted by date.

Reply:
Yeah, I saw that in the routes, it seems like a lot of extra logic now. So what is the point of having the ItemSchema now? It seems like the BatchSchema has everything we need. I had originally wanted the ItemSchema to be able to keep all the batches of the same item together, but if we are not doing that anymore, is the ItemSchema still necessary?

Reply:
Currently, the `ItemSchema` tracks the `itemId` as well, but the `BatchSchema` does not. If we put the `itemId` in each batch as well, then we wouldn't need `ItemSchema` anymore. However, I feel like it makes sense to have `ItemSchema` for item-specific information, especially if we want to add more item-specific fields later. For example, the project proposal mentions having an image for each item, so if we do that, it would be easier to implement with a separate `ItemSchema`. What do you think?

Reply:
Okay that makes sense, thank you for explaining!